packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-enabled.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Icon, ThemedScrollbars, Text } from '@deriv/components';
+import DigitForm from './digit-form';
+import { isMobile } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+const TwoFactorEnabled = () => {
+    return (
+        <ThemedScrollbars is_bypassed={isMobile()} className='two-factor__scrollbars'>
```

Comment:
can we use `is_mobile` from `ui` store here?

Reply:
I think let's incorporate it once the latest changes for is_mobile are pushed to master?